src/lib/tenant/tenantUI.ts

```diff
@@ -65,6 +65,19 @@ type TenantUIParams = {
   toggles?: UIToggle[];
   governanceIssues?: UIGovernanceIssue[];
   governanceStakeholders?: UIGovernanceStakeholder[];
+  customization?: {
+    primary?: string;
+    secondary?: string;
+    tertiary?: string;
+    neutral?: string;
+    wash?: string;
+    line?: string;
+    positive?: string;
+    negative?: string;
+    brandPrimary?: string;
+    brandSecondary?: string;
+    font?: string;
+  };
```

Comment:
@andreitr would be good to get your feedback here if you think this theming setup thought tenants makes sense

Reply:
I think this makes sense. It would be good to give this template to Josiah so he can do this along with tenant assets.